code/modules/maps/_map_template.dm

```
@@ -154,12 +156,8 @@
 /datum/map_template/proc/load(turf/T, centered=FALSE)
 	if(centered)
 		T = locate(T.x - round(width/2) , T.y - round(height/2) , T.z)
-	if(!T)
-		return
-	if(T.x+width > world.maxx)
-		return
-	if(T.y+height > world.maxy)
-		return
+	if(!T || ((T.x + width) > world.maxx) || ((T.y + height) > world.maxy))
```

Null `T` is causing the CRASH messages to fail on a null reference.

---

Yeah, but that's legit. It would silently return on null T anyways before.

---

I know what you mean now. The crash string prints out the turf's coordinates without nullchecking them..